frontend/src/ui/Button.ts

```diff
@@ -48,9 +49,9 @@ const ButtonBase = styled.button<{ full?: boolean }>`
 
   ${mediumWeight};
 
-  ${media.min('tablet')`
+  @media (min-width: ${BREAKPOINTS['tablet']}) {
```

Comment:
Probably make sense to apply this to the whole project after this PR gets merged.